src/main/java/com/solvd/pages/HomePage.java

```diff
@@ -0,0 +1,47 @@
+package com.solvd.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class HomePage {
+    WebDriver driver;
+
+    @FindBy(css = "a[class*='menu_home'][class*='active']")
+    WebElement homeButton;
+
+    @FindBy(id = "main_menu")
+    WebElement homeMenu;
+
+    @FindBy(id = "filter_keyword")
+    WebElement searchInput;
+
+    @FindBy(className = "button-in-search")
+    WebElement searchButton;
+
+    public HomePage(WebDriver driver) {
+        driver.get("https://automationteststore.com/");
```

Comment:
driver.get() inside a Page Object is not recommended, navigation should be handled externally. 
public void navigateToHomePage() {
        driver.get("https://automationteststore.com/");
    }